seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.common.constants;
+
+public enum JobMode {
+    BATCH, STREAMING, STRUCTURED_STREAMING
```

Comment:
We can involve more people in the discussion，@[CalvinKirs](https://github.com/CalvinKirs)

Reply:
Yes, it can compile success.

Reply:
The ci is failed, you can rebase from master then this will be solved.

Reply:
CI failure has nothing to do with this PR, just need to update the code 🤣

Reply:
CI failure has nothing to do with this PR, just need to update the code 🤣

Reply:
spark structured streaming and spark dstream cannot be initialized at the same time

Reply:
Since this PR has been put on hold for a long time, and there is no plugin for spark stream, I will merge it first, and then solve the problem here. Thank you for your reminder.